platform/app/public/config/default.js

```diff
@@ -1,4 +1,5 @@
 window.config = {
+  disableEditing: true,
```

Comment:
can you remove this please

Reply:
we should add it to the documentation in the configuration though

Reply:
Ok